modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -0,0 +1,65 @@
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemIcon from '@elementor/ui/ListItemIcon';
+import ListItemText from '@elementor/ui/ListItemText';
+import Collapse from '@elementor/ui/Collapse';
+import ChevronDownIcon from '@elementor/icons/ChevronDownIcon';
+import RadioButtonUncheckedIcon from '@elementor/icons/RadioButtonUncheckedIcon';
+import CardMedia from '@elementor/ui/CardMedia';
+import Link from '@elementor/ui/Link';
+
+function CheckListItem( props ) {
+	const { id, title, description, link, CTA } = props.step,
+		[ expanded, setExpanded ] = React.useState( false );
+
+	const handleExpandClick = () => {
+		setExpanded( ! expanded );
+	};
+
+	return (
+		<>
+			<ListItemButton onClick={ () => handleExpandClick( id ) } sx={ {
+				'&.MuiButtonBase-root:hover': {
+					bgcolor: 'transparent',
+				},
+			} }>
+				<ListItemIcon> <RadioButtonUncheckedIcon /> </ListItemIcon>
+				<ListItemText id={ title } primary={ <Typography variant="body2">{ title }</Typography> } />
+				{ expanded ? <ChevronDownIcon sx={ { transform: 'rotate(180deg)' } } /> : <ChevronDownIcon /> }
+			</ListItemButton>
```

Comment:
This should be a standalone component in my opinion (i.e. checklist-item-button)

Reply:
don't want to split, will have to pass props through another level down in 2 components